opencti-platform/opencti-front/src/utils/filters/filtersUtils.tsx

```diff
@@ -282,6 +284,90 @@ export const filtersWithEntityType = (
   };
 };
 
+// remove filter with key=entity_type and values contains 'all'
+// because in this case we want everything, so no need for filters
+export const removeAllTypesFromFilter = (inputFilters?: FilterGroup) => {
```

Comment:
function names is misleading, we naively understand that it removes all filters on entity_types, not the 'all' special case.

```suggestion
export const removeEntityTypesAllFromFilter = (inputFilters?: FilterGroup) => {
```

Reply:
done